dashpay/src/main/kotlin/org/dashevo/dashpay/Profiles.kt

```diff
@@ -29,15 +33,43 @@ class Profiles(
         signingKey: ECKey
     ) {
         val profileDocument = createProfileDocument(displayName, publicMessage, avatarUrl, identity)
+        profileDocument.createdAt = Date().time
 
         val transitionMap = hashMapOf(
             "create" to listOf(profileDocument)
         )
 
+        signAndBroadcast(transitionMap, identity, id, signingKey)
+    }
+
+    fun replace(
+        displayName: String,
+        publicMessage: String,
+        avatarUrl: String?,
+        identity: Identity,
+        id: Int,
+        signingKey: ECKey
+    ) {
+        val profileDocument = createProfileDocument(displayName, publicMessage, avatarUrl, identity)
+        profileDocument.updatedAt = Date().time
+
+        val transitionMap = hashMapOf(
+            "replace" to listOf(profileDocument)
+        )
+
+        signAndBroadcast(transitionMap, identity, id, signingKey)
+    }
```

Comment:
This method will allow a profile to be updated, but it has not been tested.

Reply:
This probably doesn't work, but will be changed in a future PR when profiles are fixed for creating and updating/replacing.